nilearn/_utils/data_gen.py

```diff
@@ -102,78 +156,98 @@ def generate_regions_ts(n_features, n_regions,
     return regions
 
 
-def generate_maps(shape, n_regions, overlap=0, border=1,
-                  window="boxcar", rand_gen=None, affine=np.eye(4)):
+def generate_maps(shape,
+                  n_regions,
+                  overlap=0,
+                  border=1,
+                  window="boxcar",
+                  random_state=0,
+                  affine=np.eye(4)):
     """Generate a 4D volume containing several maps.
 
     Parameters
     ----------
-    n_regions : int
+    n_regions : :obj:`int`
         Number of regions to generate.
 
-    overlap : int, optional
+    overlap : :obj:`int`, optional
         Approximate number of voxels common to two neighboring regions.
         Default=0.
 
-    window : str, optional
+    window : :obj:`str`, optional
         Name of a window in scipy.signal. Used to get non-uniform regions.
         Default='boxcar'.
 
-    ran_gen : None or numpy.random.RandomState, optional
-        Random generator to use for generation. Default=None.
+    random_state : :obj:`int` or :obj:`numpy.random.RandomState` instance, \
+                   optional
+        Random number generator, or seed.
+        Default=0.
 
-    affine : Numpy array, optional
-        Affine transformation to use. Default=np.eye(4).
+    affine : :obj:`numpy.ndarray`, optional
+        Affine transformation to use.
+        Default=np.eye(4).
 
-    border : int, optional
+    border : :obj:`int`, optional
         Number of background voxels on each side of the 3D volumes.
         Default=1.
 
     Returns
     -------
-    maps : nibabel.Nifti1Image
-        4D array, containing maps.
+    maps : Niimg-like object
+        4D array containing maps.
```

Comment:
The description implies a numpy array to me. What about `4D image object containing the maps.`?